blueprints/data-solutions/data-platform-minimal/variables.tf

```diff
@@ -24,6 +24,16 @@ variable "composer_config" {
       env_variables            = optional(map(string), {})
       image_version            = optional(string, "composer-2-airflow-2")
     }), {})
+    web_server_network_access_ip_ranges = optional(list(map(string)), [
```

Comment:
- I'd keep the original name from the provider `web_server_network_access_control` or maybe even `web_server_access_control`.
- For the type I think `optional(map(string))` without any default a better option. You can use it with something like  
  ```terraform
  web_server_access_control = { 
    "10.0.0.0/8" = "Allow 10/8"
    "192.168.0.0/16" = "Allow 192.168/16"
  } 
  ```

Reply:
cool
- I'd used a list of maps to support multiple ranges (which composer supports)
```
web_server_network_access_ip_ranges = [
  {
    description = "Range 1"
    value = "1.2.3.4/32"
  },
  {
    description = "Range 2"
    value = "4.5.6.7/32"
  }
]
```
- I provided a default that matches the actually composer default so that if you remove your custom ranges it reverts to the defaults, I'll double check that it doesn't just revert to them anyway.

Finishing up some more testing of the edge cases

Reply:
Ah, i'll try a single map, thanks

the default for a composer web server is `public` so replicating that for the default is ok?

can you explain the remaining linting error? couldn't make heads or tails of the failure message, cheers!

Reply:
> Ah, i'll try a single map, thanks. so the for_each will stay the same?

The for_each yes, but you'll probably have to change the content of the dynamic block

> the default for a composer web server is public so replicating that for the default is ok?

I'd rather no. Just leave it without default.

> can you explain the remaining linting error? couldn't make heads or tails of the failure message, cheers!

You have to update the variables table. We have a script to do that, check our [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#quick-developer-workflow).